protocols/virtualfund/virtualfund.go

```diff
@@ -344,9 +344,9 @@ func (s VirtualFundObjective) generateLedgerRequestSideEffects() []protocols.Led
 				ObjectiveId: s.Id(),
 				LedgerId:    s.ToMyRight.Channel.Id,
 				Destination: s.V.Id,
-				Left:        s.ToMyRight.Channel.MyDestination(),
+				Left:        types.AddressToDestination(s.ToMyRight.Channel.Participants[0]),
 				LeftAmount:  leftAmount,
-				Right:       s.ToMyRight.Channel.TheirDestination(),
+				Right:       types.AddressToDestination(s.ToMyRight.Channel.Participants[1]),
```

Comment:
I'm not sure this is correct -- I think it should depend on `myRole` ?

Reply:
It works out as correct _if_ we have a convention on the ordering of participants in ledger channels. But if you check the SATP paper, I don't believe this is specified. What _is_ specified is the role of the participants in the virtual fund protocol*.

So I'm nitpicking a bit, but I think it is worth fixing this because it allows ledger channels to be spun up with participants in any order (that might equate to either party -- end user or hub -- proposing the ledger channel). 

Here's my proposed fix https://github.com/statechannels/go-nitro/pull/288. 

* We can take this to be the same as the index of each participant in the participant list in the virtual channel.

Reply:
This nitpick is very worthwhile. It is not good to couple the participant ordering of the virtual channel `V`, the ledger channels, AND the guarantees.